.github/workflows/deployments.yml

```diff
@@ -537,7 +511,7 @@ jobs:
             amd64_image=$(echo "$EXT_JSON" | jq -r ".image_hash[\"$key\"]")
             echo "CUDA $cuda amd64 image: $amd64_image"
 
-            stitched_tag="ghcr.io/nvidia/cuda-quantum-devdeps:cu${cuda}-ext-${branch}"
+            stitched_tag="ghcr.io/nvidia/cuda-quantum-devdeps:ext-cu${cuda}-gcc11-${branch}"
```

Comment:
Please add this to the list of follow up items: The gcc11 hardcoded here is also something that would be nice to get from prior jobs instead.